controllers/scc.go

```diff
@@ -0,0 +1,169 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
+
+import (
+	"context"
+	"fmt"
+
+	secv1 "github.com/openshift/api/security/v1"
+	lvmv1alpha1 "github.com/red-hat-storage/lvm-operator/api/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	sccName = "topolvm-scc"
+)
+
+type openshiftSccs struct{}
+
+// openshiftSccs unit satisfies resourceManager interface
+var _ resourceManager = openshiftSccs{}
+
+func (c openshiftSccs) getName() string {
+	return sccName
+}
+
+func (c openshiftSccs) ensureCreated(r *LVMClusterReconciler, ctx context.Context, lvmCluster *lvmv1alpha1.LVMCluster) error {
+	if r.IsOpenshift == true {
+
+		sccs := getAllSCCs(lvmCluster.Namespace)
+		for _, scc := range sccs {
+			found, err := r.SecurityClient.SecurityContextConstraints().Get(scc.Name, metav1.GetOptions{})
+			if err != nil && errors.IsNotFound(err) {
+				r.Log.Info("Creating SecurityContextConstraint.", "SecurityContextConstraint", scc.Name)
+				_, err := r.SecurityClient.SecurityContextConstraints().Create(scc)
+				if err != nil {
+					return fmt.Errorf("unable to create SCC %+v: %v", scc, err)
+				}
+			} else if err == nil {
+				scc.ObjectMeta = found.ObjectMeta
+				r.Log.Info("Updating SecurityContextConstraint.", "SecurityContextConstraint", scc.Name)
+				_, err := r.SecurityClient.SecurityContextConstraints().Update(scc)
+				if err != nil {
+					r.Log.Error(err, "Unable to update SecurityContextConstraint.", "SecurityContextConstraint", scc.Name)
```

Comment:
```suggestion
					r.Log.Error(err, "failed to update SecurityContextConstraint.", "SecurityContextConstraint", scc.Name)
```
just to be consistent with other error messages.